tools/make_release_tarball

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+if [ ! -e tools/make_release_tarball  -o  "$1" != "" ]
+then
+  echo "Handy script for creating tarball: Use from the checkout directory"
+  exit 0
+fi
+
+set -e
+
+VERSION_MAJOR=$(grep set.MIR_VERSION_MAJOR CMakeLists.txt | cut -f2 -d' ' | tr -d \))
+VERSION_MINOR=$(grep set.MIR_VERSION_MINOR CMakeLists.txt | cut -f2 -d' ' | tr -d \))
+VERSION_PATCH=$(grep set.MIR_VERSION_PATCH CMakeLists.txt | cut -f2 -d' ' | tr -d \))
+
+VERSIONED_NAME=mir-$VERSION_MAJOR.$VERSION_MINOR.$VERSION_PATCH
```

Comment:
Think we could use `git describe --tags` here instead?

Reply:
We could, if we're going to guarantee that the release tags are there at the right time :)

Reply:
Well, let's!

Reply:
For building tarballs to upload to the staging PPA we likely want to take the version from CMakeLists.txt.

Also, the guarantee doesn't (yet) hold:

$ git describe --tags
v0.27.0-614-g8f4e0403c8

Reply:
Well, the version from CMakeLists.txt is not enough, since there's code on top of it.

Anyway, let's discuss on the forums (will post soon).